src/modules/core/db.logger.js

```diff
@@ -0,0 +1,54 @@
+const {
```

Comment:
what is the difference between this file and src/modules/core/database.logger.js?

Reply:
They are used for different db instance,currently are all for MySQL and no difference